regression/goto-instrument/dump-non-tag-struct/main.c

```diff
@@ -0,0 +1,6 @@
+#include <stdlib.h>
+
+int main()
+{
+  calloc(10, 1);
```

Comment:
I don't quite get why this creates a structure rather than an array?

Reply:
Comment added.